index.js

```diff
@@ -111,11 +125,46 @@ const connect = async () => {
 
         return producer;
     } catch (err) {
-        logger.error('kafka broker connection failiure', err);
+        logger.error('kafka broker connection failure', err);
         throw err;
     }
 };
 
+/**
+ * Connects to a Kafka client instance as admin
+ * @returns Admin promise object
+ */
+const connectAdmin = async () => {
+    const admin = connect(true);
+
+    logger.info('kafka Broker Admin connected');
+
+    errorTypes.map(type => {
```

Comment:
Since we are not using the resulting array from `.map`
```suggestion
    errorTypes.forEach(type => {
```